src/Eshopworld.WorkerProcess/LeaseAllocator.cs

```diff
@@ -34,83 +35,103 @@ public class LeaseAllocator : ILeaseAllocator
         /// <inheritdoc />
         public async Task<ILease> AllocateLeaseAsync(Guid instanceId)
         {
+            //Lease allocation following design documented here : https://eshopworld.atlassian.net/wiki/spaces/EPLT/pages/2345435290/Worker+Process
```

Comment:
I suggest we keep it in the PR description rather than code

Reply:
Removed that comment